src/components/FAB/FAB.tsx

```diff
@@ -188,7 +190,7 @@ const FAB = ({
 
   const rippleColor = color(foregroundColor).alpha(0.32).rgb().string();
 
-  return (
+  return showFAB ? (
```

Comment:
Nit: using `&&` should be enough here I think, so can avoid the second half of the ternary.

Reply:
I tried using `&&` originally but got type errors on the `withTheme(FAB)` lines at the bottom of the file. Using the ternary prevented the type errors so I went with that instead.

Reply:
Fair! I didn't go that far to test it 😅